packages/node-opentelemetry/__tests__/otel.test.ts

```diff
@@ -1,6 +1,11 @@
-import { initSDK } from '../src/otel';
+import { initSDK, shutdown } from '../src/otel';
 
 describe('otel', () => {
+  it('can sucessively shutdown without initialization', () => {
+    shutdown();
```

Comment:
Is this intentional (to call it 2 times) ?

Reply:
yeah the test is testing that it can be called, and successively safely as well